### ISSUE TITLE
The obtained FID results are far away from the paper.

### ISSUE THREAD

**Issue by ThisisBillhe** [duchenzhuang/FSQ-pytorch#4]
Hi, thanks for your code! 
I trained FSQ model ([8,5,5,5]) on ImageNet 128x128 for about 100ksteps, and the loss is almost not decreasing. However, I got a results of :
```
fid score tensor(51.6240, device='cuda:0')
l1loss: 0.226502833597343
precep_loss: 0.2657754932294416
codebook usage 1.0
```
The FID is far away from the paper, which is about 8 according to Figure 3(a) in the paper. Any idea how to reproduce the FID results?

**Comment by Doraemonzzz** [comment]
I have reproduced similar results as you, and I think the main differences may come from two places: the network structure and training hyperparameters such as update steps.
